app/swagger/swagger/requests/mdot/supplies.rb

```diff
@@ -26,43 +27,60 @@ class Supplies
-                  property :state, type: :string
-                  property :country, type: :string
-                  property :postal_code, type: :string
-                end
 
+                property :use_permanent_address, type: :boolean, example: true
+                property :use_temporary_address, type: :boolean, example: false
+                property :vet_email, type: :string, example: 'vet1@va.gov'
                 property :order do
                   key :type, :array
 
                   items do
                     key :type, :object
-
-                    property :product_id, type: :string
+                    property :productId, type: :integer, example: 2499
                   end
                 end
+
+                property :permanent_address do
+                  key :type, :object
+                  property :isMilitary, type: :boolean, example: false
+                  property :street, type: :string, example: '125 SOME RD'
+                  property :street2, type: :string, example: 'APT 101'
+                  property :city, type: :string, example: 'DENVER'
+                  property :state, type: :string, example: 'CO'
+                  property :country, type: :string, example: 'United States'
+                  property :postalCode, type: :string, example: '11111'
+                end
+
+                property :temporary_address do
+                  key :type, :object
+                  property :isMilitary, type: :boolean, example: false
+                  property :street, type: :string, example: '17250 w colfax ave'
+                  property :street2, type: :string, example: 'a-204'
+                  property :city, type: :string, example: 'Golden'
+                  property :state, type: :string, example: 'CO'
+                  property :country, type: :string, example: 'United States'
+                  property :postalCode, type: :string, example: '80401'
+                end
               end
             end
 
             response 200 do
-              key :description, 'mdot order response'
-
+              key :description, 'Response is OK'
               schema do
-                key :required, %i[status order_id]
-
-                property :status, type: :string
-                property :order_id, type: :string
+                key :type, :array
+                items do
+                  key :type, :object
+                  property :productId, type: :integer, example: 2499
+                  property :orderID, type: :integer, example: 1001
+                  property :status,
+                           type: :string,
+                           example: 'Order Processed',
+                           enum: [
+                             'Order Processed',
+                             'Order Pending',
+                             'Unable to order item since the last order was less than 5 months ago.'
+                           ]
+                end
```

Comment:
The [olive_branch gem](https://github.com/vigetlabs/olive_branch) library should take care of the translation of camelCase properties to snake_case. Since we're working in ruby, which uses snake_case variable names, we should keep these snake_case, I believe.

Reply:
Thanks @radavis, updated to use snake_case